streams/src/test/java/org/apache/kafka/streams/processor/internals/StateDirectoryTest.java

```diff
@@ -88,27 +89,6 @@ public void shouldCreateTaskStateDirectory() {
         assertTrue(taskDirectory.isDirectory());
     }
 
-    @Test
-    public void shouldLockTaskStateDirectory() throws IOException {
```

Comment:
Why did you remove this test?

Reply:
directory.lock(taskId) is at different level than channel.tryLock() , rendering the test useless.